modules/tinymce/src/themes/silver/test/ts/browser/skin/OxideCollectionComponentTest.ts

```diff
@@ -6,105 +6,102 @@ import { Attribute, SugarBody, SugarDocument, SugarElement } from '@ephox/sugar'
+      return Arr.get(matches, n).getOrDie(`Could not find match ${n} of selector: ${selector}`);
+    };
 
-  const findNthIn = (selector: string, n: number, elem: SugarElement<Node>) => {
-    const matches = UiFinder.findAllIn(elem, selector);
-    return Arr.get(matches, n).getOrDie(`Could not find match ${n} of selector: ${selector}`);
-  };
+    const hook = TinyHooks.bddSetup<Editor>({
+      toolbar: 'dialog-button',
+      base_url: '/project/tinymce/js/tinymce',
+      setup: (ed: Editor) => {
+        ed.ui.registry.addButton('dialog-button', {
+          type: 'button',
+          text: 'Go',
+          onAction: () => {
+            ed.windowManager.open({
+              title: 'Testing list component',
+              initialData: {
+                'stuff-1': [
+                  { value: 'a', text: 'text-A', icon: 'icon-A' },
+                  { value: 'b', text: 'text-B', icon: 'icon-B' },
+                  { value: 'b', text: 'text-C', icon: 'icon-C' }
+                ],
+
+                'stuff-auto': [
+                  { value: 'd', text: 'text-D', icon: 'icon-D' },
+                  { value: 'e', text: 'text-E', icon: 'icon-E' },
+                  { value: 'f', text: 'text-F', icon: 'icon-F' }
+                ],
+
+                'stuff-2': [
+                  { value: 'g', text: 'text-G', icon: 'icon-G' },
+                  { value: 'h', text: 'text-H', icon: 'icon-H' },
+                  { value: 'i', text: 'text-I', icon: 'icon-I' }
+                ],
+                'input1': ''
+              },
+              body: {
+                type: 'panel',
+                items: [
+                  {
+                    name: 'input1',
+                    type: 'input'
+                  },
+                  {
+                    name: 'stuff-1',
+                    type: 'collection',
+                    columns: 1
+                  },
+                  {
+                    name: 'stuff-auto',
+                    type: 'collection',
+                    columns: 'auto'
+                  },
+                  {
+                    name: 'stuff-2',
+                    type: 'collection',
+                    columns: 2
+                  }
+                // TODO: TINY-3229 implement collection columns properly
+                ] as any[]
+              },
+              buttons: []
+            });
+          }
+        });
+      }
+    }, []);
 
-  context('Check structure of collection in a dialog', () => {
     TestHelpers.GuiSetup.bddAddStyles(SugarDocument.getDocument(), [
       ':focus { outline: 2px solid green; }'
     ]);
 
-    before(async () => {
-      const editor = hook.editor();
-      TinyUiActions.clickOnToolbar(editor, 'button');
-      await TinyUiActions.pWaitForDialog(editor);
+    before(function () {
+      this.skip();
+      // const editor = hook.editor();
+      // TinyUiActions.clickOnToolbar(editor, 'button');
+      // await TinyUiActions.pWaitForDialog(editor);
```

Comment:
Ah right, I missed that it was in the existing file. Although it looks some new comments were added which might have been what confused me
```suggestion
			// TODO: TINY-3229 implement collection columns properly
      this.skip();
```